src/Akka.Persistence.Sql/Snapshot/ByteArraySnapshotDao.cs

```diff
@@ -182,41 +416,50 @@ await connection
                 });
         }
 
-        public async Task<Option<SelectedSnapshot>> LatestSnapshotAsync(
+        public Task<Option<SelectedSnapshot>> LatestSnapshotAsync(
             string persistenceId,
             CancellationToken cancellationToken = default)
         {
-            var cts = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken, _shutdownCts.Token);
```

Comment:
This should probably pass the CTS or whatever into the request, for the sake of allowing the token to do the needful.